infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/kubernetes-external-secrets.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: kubernetes-external-secrets
+  namespace: kubernetes-external-secrets
+  labels:
+    app: kubernetes-external-secrets
+spec:
+  selector:
+    app: kubernetes-external-secrets
+  ports:
+    - protocol: TCP
+      port: 3001
+      name: prometheus
+      targetPort: prometheus
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubernetes-external-secrets
+  namespace: kubernetes-external-secrets
+  labels:
+    app: kubernetes-external-secrets
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubernetes-external-secrets
+  template:
+    metadata:
+      labels:
+        app: kubernetes-external-secrets
+    spec:
+      serviceAccountName: kubernetes-external-secrets
+      containers:
+        - name: kubernetes-external-secrets
+          image: "ghcr.io/external-secrets/kubernetes-external-secrets:7.0.1"
+          imagePullPolicy: Always
+          ports:
+          - name: prometheus
+            containerPort: 3001
+          env:
+          - name: "LOG_LEVEL"
+            value: "debug" # TODO(ameukam): switch to info
```

Comment:
Comment can be removed. I think we should leave `debug` as the default log level.